### PR TITLE
Feature/core 541 permissions ui

### PR DESCRIPTION
**Pull request by karladler**
**Update: can be merged yet, after approve!**

### CODE REVIEW

File: dapps/evancore.vue.libs/src/i18n/de.ts [hunk 1]
```diff
@@ -107,16 +107,25 @@ export default {
...
     },
+    "read": "lesen",
     "save": "Speichern",
     "share": "Freigeben",
+    "sharing": {
+      "update": "Freigaben aktualisieren",
+      "selectContact": "Kontakt für die Freigabe auswählen",
+      "defineFor": "Setzen Sie die Zugriffsrechte für ${contactName}.",
+      "permissionsTitle": "Zugriffsrechte",
+      "noContacts": "Sie haben keine Kontakte in ihrem Adressbuch, bitte fügen Sie zuerst einen Kontakt hinzu."
```
Comment (wulfraem) [inline dapps/evancore.vue.libs/src/i18n/de.ts]:
```suggestion
      "noContacts": "Sie haben keine Kontakte in Ihrem Adressbuch, bitte fügen Sie zuerst einen Kontakt hinzu."
```

**Review by wulfraem — CHANGES_REQUESTED** [review]
^^

File: dapps/evancore.vue.libs/src/components/permissions-editor/permissions-editor.ts [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+import Vue from 'vue';
+import Component, { mixins } from 'vue-class-component';
+
+// evan.network imports
+import { deepEqual } from '@evan.network/ui';
+import EvanComponent from '../../component';
+
+import { Prop } from 'vue-property-decorator';
+
+interface PermissionsInterface {
```
Comment (DanielMaass) [inline dapps/evancore.vue.libs/src/components/permissions-editor/permissions-editor.ts]: interface should define in one place. Maybe in permissions.ts

File: dapps/evancore.vue.libs/src/components/permissions-editor/permissions-editor.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <h3>Update Permissions</h3>
+    <p>
+      To share your profile information with another contact, fill out the form below and click on
+      “Share Profile Data”
+    </p>
```
Comment (DanielMaass) [inline dapps/evancore.vue.libs/src/components/permissions-editor/permissions-editor.vue]: we should support localization

File: dapps/evancore.vue.libs/src/components/permissions/permissions.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+// evan.network imports
+import EvanComponent from '../../component';
+
+import { Prop } from 'vue-property-decorator';
+
+interface PermissionsInterface {
+  [property: string]: {
+    read: boolean,
+    readWrite: boolean,
+    fields: string[]
```
Comment (DanielMaass) [inline dapps/evancore.vue.libs/src/components/permissions/permissions.ts]: maybe define as optional "fields?: string[]"

File: dapps/evancore.vue.libs/src/components/permissions/permissions.ts [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+@Component({ })
+class Permissions extends mixins(EvanComponent) {
+  readAll = false;
+  readWriteAll = false;
+
+  /**
+   * The dataset name to be displayed.
+   */
+  @Prop({
+    default: ''
```
Comment (DanielMaass) [inline dapps/evancore.vue.libs/src/components/permissions/permissions.ts]: set as required as well

File: dapps/evancore.vue.libs/src/components/permissions/permissions.vue [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+              :id="`${dataSetId}-${property}-write`"
+              :value="permissions[property].readWrite"
+              @input="val => setReadWrite(property, val)"
+            />
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <div v-else>
+      <p class="p-6 text-error">No permisisons available to edit.</p>
```
Comment (DanielMaass) [inline dapps/evancore.vue.libs/src/components/permissions/permissions.vue]: support localization

**Review by DanielMaass — APPROVED** [review]

**Pull request merged**